ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -43,6 +43,19 @@ def AddV2(*, x, y, name="AddV2"):
     return ivy.addv2(tf_frontend.to_tensor(x), tf_frontend.to_tensor(y))
 
 
+@to_ivy_arrays_and_back
+def ApproximateEqual(
+    *,
+    x,
+    y,
+    tolerance=1e-05,
+    name="ApproximateEqual",
+):
+    x, y = check_tensorflow_casting(x, y)
+    ret = ivy.abs(x-y)
```

Comment:
the lint failures may be caused by this, it needs spaces in between like `x - y`, but this is not a major issue as I think we have a workflow for fixing this, feel free to push a fix though :)